src/test/resources/bags/ddm-incorrect-urls/metadata/dataset.xml

```diff
@@ -21,6 +21,8 @@
         <ddm:references scheme="URL">xttp://abc.def</ddm:references>
         <ddm:references scheme="DOI">99.1234.abc</ddm:references>
         <ddm:references scheme="URN">uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66</ddm:references>
```

Comment:
@lindareijnhoudt these were previous examples, but I doubt whether these schemes without prefix are actually valid. Can you look into that?

Reply:
As discussed offline with @lindareijnhoudt, this is correct. Both `scheme="URN"` and `scheme="id-type:URN"` are possible according to the schema.